nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/CSVReader.java

```diff
@@ -49,7 +51,7 @@
     + "the values. See Controller Service's Usage for further documentation.")
 public class CSVReader extends SchemaRegistryService implements RecordReaderFactory {
 
-    private final AllowableValue headerDerivedAllowableValue = new AllowableValue("csv-header-derived", "Use String Fields From Header",
+    static final AllowableValue HEADER_DERIVED_ALLOWABLE_VALUE = new AllowableValue("csv-header-derived", "Use String Fields From Header",
```

Comment:
Making this static (and capitalizing the name) aligns with the common constant / property pattern, thanks! However it also looks like it can remain private (at least that's what IntelliJ tells me ;)